src/main/webapp/app/exercises/shared/feedback/feedback.component.ts

```diff
@@ -154,6 +154,9 @@ export class FeedbackComponent implements OnInit {
         if (!this.exerciseType && isProgrammingExerciseParticipation(this.result?.participation)) {
             this.exerciseType = ExerciseType.PROGRAMMING;
         }
+
+        this.showTestDetails =
+            this.exercise?.isAtLeastTutor || (this.exerciseType === ExerciseType.PROGRAMMING && (this.exercise as ProgrammingExercise)?.showTestNamesToStudents) || false;
```

Comment:
The PR description refers to editors, here you have tutors. What is the correct role to see the test names?

Reply:
See the discussion with @JohannesStoehr above. I initially only changed it to editors, but some pages like the assessment code editor were already showing test case names to tutors. So I decided to show test case names also to tutors consistently.